### DISCUSSION TITLE
keymapping for so2 (standoff2)

### DISCUSSION THREAD

**Discussion by daivrot** [barry-ran/QtScrcpy#886] [Q&A] [answered]
please give me so2 keymap default setting

**Comment by w4po** [comment, marked as answer]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper
You can now create your own

**Reply by aspect-cell** [reply]
Hey can u give me k2er app keymap code for standoff 2 keyboard and mouse ,Thank you

**Comment by aspect-cell** [comment]
Pls someone give code for k2er standoff 2 keymapping keyboard and mouse 🙏🙏
